python/tests/variable_creation_test.py

```diff
@@ -230,11 +230,11 @@ def test_array_creates_correct_variable():
     assert sc.identical(var, expected)
 
 
-def test_array_needs_nonempty_dims():
-    with pytest.raises(ValueError):
-        sc.array(dims=[], values=[])
-    with pytest.raises(ValueError):
-        sc.array(dims=None, values=[])
+def test_array_empty_dims():
+    assert sc.identical(sc.array(dims=[], values=[1]), sc.scalar([1]))
```

Comment:
Does this make a variable of `dtype=PyObject` (list) or does it extract the list element? The latter would be inconsistent with numpy.